packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -153,16 +153,18 @@ const CFDRealAccountDisplay = ({
         }
     };
 
-    const existing_accounts_data = (acc_type: 'synthetic' | 'financial') => {
+    const existing_accounts_data = (acc_type: 'synthetic' | 'financial' | 'all') => {
```

Comment:
Since our new DerivX accounts would be just "all" type, you kept the synthetic and financial for backward compatibility ?

Reply:
Yes, this is temporary backward compatibility before [this](https://github.com/binary-com/deriv-app/pull/6636) PR gets merged.